packages/babel-plugin/src/__tests__/module-traversal.test.tsx

```diff
@@ -530,18 +574,10 @@ describe('module traversal', () => {
 
         const color = { blue: 'blue' };
 
-        const Component = (props) => {
-          return <div css={{ ...objects.colorMixin2(color.blue) }} />
-        };
-    `);
+        <div css={{ ...objects.colorMixin2(color.blue) }} />
+      `);
 
-      expect(result).toIncludeMultiple([
-        '{color:red}',
-        // We're not handling statically evaluating param values used from a namespace chain,
-        // as the long term strategy is to only statically evaluate expressions scoped locally
-        // to the current file.
-        '{background-color:var(--_1w9snyi)}',
```

Comment:
Was able evaluate this statically by treating call expressions like `objects.colorMixin2(color.blue)` as a member expression